public/app/plugins/datasource/influxdb/components/InfluxLogsQueryField.tsx

```diff
@@ -133,6 +133,7 @@ export class InfluxLogsQueryField extends React.PureComponent<Props, State> {
     const { datasource } = this.props;
     const { measurements, measurement, field, error } = this.state;
     const cascadeText = getChooserText({ measurement, field, error });
+    const hasMeasurement = measurements && measurements.length > 0;
```

Comment:
Can we extract this to a memoized function? That way you can probably also add simple tests.

Reply:
Absolutely! Good point.

Reply:
@hugohaggmark I have talked about memoizing `hasMeasurments` with @aocenas and also, I implemented and tested it. Based on both, it probably doesn't make sense to use memoization. Couple reasons: 
1. After implementing it, there weren't any significant performance improvements.
2. Code was simpler without memoization, no import necessary.
3. As there is no reason for `hasMeasurments` to be global function, I am not sure what kind of tests should be implemented, as memoize-one has implemented tests for its memoization.
Therefore, as there aren't any significant performance gains, there is probably no major reasons to memoize `hasMeasurments`.  

I am going to merge this PR, as this should be fixed in 6.5,  but let me know, if there is any reason why you think we should still implement this.